### PR TITLE
Uses the resource and action naming scheme.

### PR DESCRIPTION
**Pull request by jhtimmins**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: airflow/hooks/base.py [hunk 1]
```diff
@@ -18,14 +18,12 @@
...
 import logging
 import warnings
-from typing import TYPE_CHECKING, Any, Dict, List
+from typing import Any, Dict, List
 
+from airflow.models.connection import Connection
 from airflow.typing_compat import Protocol
 from airflow.utils.log.logging_mixin import LoggingMixin
 
-if TYPE_CHECKING:
```
Comment (ashb) [inline airflow/hooks/base.py]: You probably didn't mean to revert the changes in this file.

Comment (mik-laj) [reply airflow/hooks/base.py]: Agree. See; https://github.com/apache/airflow/pull/15361

Comment (jhtimmins) [reply airflow/hooks/base.py]: Ah shoot, you're correct
